### ISSUE TITLE
Switch query to use bbox [west, south, east, north]

### ISSUE THREAD

**Issue by gmaclennan** [digidem/osm-p2p-db#24]
Currently `osm.query(q)` expects `q` to be a bounding box of the format `[[minLat,maxLat],[minLon,maxLon]]`. This is not a format used very much elsewhere and it is not documented how this is interpreted at the poles and across the meridian.

The new GeoJSON standard [RFC 7946](https://tools.ietf.org/html/rfc7946) has a clearly defined bbox in the format `[west, south, east, north]`, with [clear explanations](https://tools.ietf.org/html/rfc7946) of how this is defined at the poles and at the meridian. I think it makes more sense to use this bbox format which is more commonly used, and it matches the format expected in the OSM REST API 0.6.

This does not need to be a breaking change, as we can continue to support both formats as needed with a simple test `Array.isArray(q[0])`.

**Comment by noffle** [comment]
YES. I get confused by the current format every couple of weeks. :+1:
